src/Commands/Billable.php

```diff
@@ -204,8 +208,51 @@ protected function execute(InputInterface $input, OutputInterface $output) {
       $rows[] = ['Total', Formatter::formatDuration($total), '100%'];
     }
 
+    if ($period === static::MONTH && $input->getOption('stats')) {
+      $rows[] = new TableSeparator();
+      $rows[] = ['', 'STATS', ''];
+      $rows[] = new TableSeparator();
+      $no_weekdays_in_month = $this->getWeekdaysInMonth(date('m'), date('Y'));
+      $days_passed = $this->getWeekdaysPassedThisMonth();
+
+      $hrs_per_day = static::HOURS_PER_DAY;
+      $total_hrs = $no_weekdays_in_month * $hrs_per_day;
+      $total_billable_hrs = $total_hrs * static::BILLABLE_PERCENTAGE;
+      $billable_hrs = $billable / 60 / 60;
+      $non_billable_hrs = $non_billable / 60 / 60;
+      $completed_hrs = $billable_hrs + $non_billable_hrs;
+
+      $rows[] = ['No. Days', "$days_passed/$no_weekdays_in_month", round(100 * $days_passed / $no_weekdays_in_month, 2) . '%'];
+      $rows[] = ['Billable Hrs', "$billable_hrs/$total_billable_hrs", round(100 * $billable_hrs / $total_billable_hrs, 2)];
+      $rows[] = ['Total Hrs', "$completed_hrs/$total_hrs", round(100 * $completed_hrs / $total_hrs, 2)];
+    }
+
     $table->setRows($rows);
     $table->render();
   }
 
+  protected function getWeekdaysInMonth($m, $y) {
+    $lastday = date("t", mktime(0, 0, 0, $m, 1, $y));
```

Comment:
Are we going to add it every year? Or maybe we could have a default as well for fallback?